packages/components/transitions/src/scale-fade.tsx

```diff
@@ -98,11 +98,7 @@ export const ScaleFade = forwardRef<ScaleFadeProps, "div", false>(
       duration,
     }
 
-    isOpen = unmountOnExit ? isOpen && unmountOnExit : true
-
-    const css: CSSUIObject = {
-      w: "100%",
-    }
+    const isOpen = unmountOnExit ? rest.isOpen && unmountOnExit : true
```

Comment:
Why was it necessary to modify this section?If you don't need to fix it, please revert back to the original.

Also, this modification will cause a warning on the console because of the `isOpen` in `rest`.